build_runner/test/generate/serve_test.dart

```diff
@@ -56,11 +51,15 @@ a:file://fake/pkg/path
     });
 
     test('blocks serving files until the build is done', () async {
+      var blockContainer = <Future>[];
```

Comment:
nit: Just have a single `nextBuildBlocker` variable which you overwrite, you aren't binding the closure to the initial value just referencing this variable directly so it will pick up changes.

Reply:
Done